model/task_lifecycle.go

```diff
@@ -886,7 +886,7 @@ func getBuildStatus(buildTasks []task.Task) string {
 
 	// finished but failed
 	for _, t := range buildTasks {
-		if evergreen.IsFailedTaskStatus(t.Status) || t.Aborted {
+		if evergreen.IsFailedTaskStatus(t.Status) || t.Aborted || t.Blocked() {
```

Comment:
This is a pretty big change; if all tasks are successful, and one task is blocked, does it make sense to return failed when nothing in the build itself actually failed? I think not -- for example, you _can_ create tasks that only run if the dependency _failed_ . It seems expected in that case then that being blocked doesn't mean it's a failure.

Reply:
that makes a lot of sense
removed